tsconfig.json

```diff
@@ -5,31 +5,27 @@
 		"experimentalDecorators": true,
 		"forceConsistentCasingInFileNames": true,
 		"isolatedModules": true,
-		"noEmit": true,
+		"noEmit": false,
```

Comment:
with this off we have to use another tool to compile the code, is the package working in web with `noEmit=false`?
https://www.typescriptlang.org/tsconfig#noEmit